docs/reference/access-control-list.md

```diff
@@ -1,16 +1,18 @@
 ---
 layout: default
-title: Access Control Lists (ACL)
+title: Access Control List (ACL)
```

Comment:
Because ACL is singular, ACLs is plural, we can't have both of them "Access Control List (ACL)" or "Access Control Lists (ACLs)"
ACL (singular) is the commonly used term AFAIK (see [wiki definition](https://en.wikipedia.org/wiki/Access-control_list))

Reply:
anyway, I don't really mind using the singular or plural, just needs to be aligned :) changing to plural